manifests/e/Exiv2/Exiv2/0.28.2/Exiv2.Exiv2.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Exiv2.Exiv2
+PackageVersion: 0.28.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: exiv2-0.28.2-2019msvc64\bin\exiv2.exe
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 792.)